common4j/src/main/com/microsoft/identity/common/java/authorities/Authority.java

```diff
@@ -154,6 +155,10 @@ public static Authority getAuthorityFromAuthorityUrl(String authorityUrl) {
                 authority = new AzureActiveDirectoryB2CAuthority(authorityUrl);
             } else if (CIAM.equalsIgnoreCase(authorityTypeStr)) {
                 authority = new CIAMAuthority(authorityUrl);
+            } else if (AAD_NA.equalsIgnoreCase(authorityTypeStr) && configuredAuthority instanceof NativeAuthCIAMAuthority) {
```

Comment:
What's AADN_NA?

Reply:
Is it really a tenant inside AAD?

Reply:
and all native auth apps/users are in that one tenant?

Reply:
There will be tenants that support NativeAuth under AAD. Its a type of tenant and not a single tenant.